src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -846,7 +847,37 @@ public PdaoLoadStatistics loadToStagingTable(
           (ingestRequest.getCsvNullMarker() == null) ? "" : ingestRequest.getCsvNullMarker());
     }
     LoadJobConfiguration configuration = loadBuilder.build();
+    Job loadJob;
+
+    if (ingestRequest.getFormat() == IngestRequestModel.FormatEnum.CSV
+        && ingestRequest.isCsvAddRowIds()) {
+      // Ingest without the datarepo_row_id column
+      Schema noRowIds = buildSchema(targetTable, false);
+      loadBuilder.setSchema(noRowIds);
```

Comment:
It's a little hard (for me at least) to see that this overrides the `setSchema()` call on line 838 above. I think it would be clearer to me to only call `loadBuilder.setSchema()` once and put the other call in the `else`.  If you do that, maybe also move `schema` down to just above the `if`, and change it from `schema` to `schemaWithRowId` ?

Also I think `noRowId` would be better than `noRowIds` since it's a schema that's missing a single column, not a table that's missing a set of row ID data.